mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -262,9 +263,10 @@ class SearchlightHyperalignment(ClassWithCollections):
             If None, hyperalignment is performed at every voxel (default).""")
 
     hyperalignment = Parameter(
-        Hyperalignment(ref_ds=0),
+        Hyperalignment(ref_ds=None),
         doc="""Hyperalignment instance to be used in each searchlight sphere.
             Default is just the Hyperalignment instance with default parameters.
+            Its `ref_ds` parameter would be overridden and therefore has no effect.
```

Comment:
Essentially we just need one `ref_ds` for everything.

We want to use the same `ref_ds` for both the `SearchlightHyperalignment` instance and the `Hyperalignment` instance it's using for hyperaligning local voxels/nodes.

Therefore we just set the `ref_ds` parameter of `Hyperalignment` with the `ref_ds` parameter of `SearchlightHyperalignment` while running, no matter what that parameter was.